numpy/core/tests/test_arrayprint.py

```diff
@@ -397,6 +399,15 @@ def test_wide_element(self):
             "[ 'xxxxx']"
         )
 
+    @given(hynp.from_dtype(np.dtype("U")))
+    def test_any_text(self, text):
+        a = np.array([text, text, text])
+        assert_equal(a[0], text)
+        assert_equal(
+            np.array2string(a, max_line_width=len(repr(text)) * 2 + 3),
+            "[{0!r} {0!r}\n {0!r}]".format(text)
+        )
+
```

Comment:
A comment with an example of what `text` is and what the test is about might help clarify what is going on here

Reply:
Done.